bin/kalite

```diff
@@ -78,6 +85,28 @@ if not os.path.exists(kalite_path):
 # Converts to normal string for Windows users, where it gets turned into a unicode string and errors out in subprocess
 env['KALITE_DIR'] = str(kalite_path)
 
+# TODO(benjaoming): Delegate responsibility to the .deb package
+if os.path.isfile("/etc/ka-lite/username"):
```

Comment:
I don't know if there's any point in assembling a path that's invalid on windows, anyways... I mean: `os.join(os.sep, "etc", ...)` is a bit odd. I think verifying that this worked was all we needed :)